### ISSUE TITLE
A copied workflow with survey task does not show images linked to survey task

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/Panoptes-Front-End#7141]
## Expected behavior
- copy workflow with survey task
- go to new workflow, survey task, "Images" tab of survey task editor
- see list of images per task object `images` property

## Current behavior
- do not see list of images per task object `images` property
- because images are not linked to new workflow resource
- to volunteer new survey task loads as expected, because urls in task object `images` property exist (as long as original workflow exists?)
- but project collaborator can't see/delete images in survey task editor "Images" tab

## Steps to replicate
- https://master.pfe-preview.zooniverse.org/lab/1634/workflows/3817?env=staging

## Additional information
- when copying a workflow, could link attached images I think [here](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/lab/actions/workflow.js#L23) with `.update({ links.attached_images: [workflowData.links.attached_images.ids] })` as per [PJC link docs](https://github.com/zooniverse/panoptes-javascript-client/tree/main/lib/json-api-client#working-with-links), or something along those lines

Linked Talk Post: https://www.zooniverse.org/talk/18/3378978

**Comment by mcbouslog** [comment]
I think the key thing that's going on is that the resources linked to the initial workflow aren't linked to the new copied workflow.

opinion - I think that's good (this issue is a PFE (lab) responsibility, not a Panoptes API responsibility). I think we should explicitly link resources as needed, it's easy to do with the PJC, and I don't think there are many resources linked to a workflow that we want copied (other than attached_images). Other than attached_images, for workflows, I think the most common linked resource would be subject_set.

**Comment by lcjohnso** [comment]
The context to my PFE vs. Panoptes question is the fact that I believe project copying is fully delegated to Panoptes. That belief was based off my memory of PR https://github.com/zooniverse/Panoptes-Front-End/pull/6212. If delegated to Panoptes, then the inclusion of workflow attached images (along with translations, but excluding subject sets) is a Panoptes issue.

I don't have strong feelings about responsibility, but want to be sure we aren't diving up the work between two apps if we can keep it centralized in one place.

**Comment by mcbouslog** [comment]
Yes I think your understanding of #6212 is correct, sounds good to transfer this to Panoptes.

Looks like this is related to zooniverse/panoptes/pull/3626 . I'm not sure if linking `attached_images` was considered. When are attached_images linked to a workflow? Is it only with a survey task?
